arrows/klv/klv_0601.h

```diff
@@ -225,6 +225,25 @@ KWIVER_ALGO_KLV_EXPORT
 std::ostream&
 operator<<( std::ostream& os, klv_0601_sensor_fov_name value );
 
+// ----------------------------------------------------------------------------
+/// Source of the navigation positioning information.
+enum KWIVER_ALGO_KLV_EXPORT klv_0601_positioning_method_source
+{
+  KLV_0601_POSITIONING_METHOD_SOURCE_ON_BOARD_INERTIAL_NAVIGATION_SYSTEM,
```

Comment:
```suggestion
  KLV_0601_POSITIONING_METHOD_SOURCE_ON_BOARD_INS,
```
INS is a common abbreviation, so we can save a few keystrokes